src/plugins/intel_cpu/src/utils/debug_caps_config.h

```diff
@@ -43,6 +43,7 @@ class DebugCapsConfig {
     // std::hash<int> is necessary for Ubuntu-16.04 (gcc-5.4 and defect in C++11 standart)
     std::unordered_map<FILTER, std::string, std::hash<int>> blobDumpFilters;
     std::string summaryPerf = "";
+    std::string snippets_err_detector;
```

Comment:
Minor: as for me, name `snippets_segfault_detector` better characterize the goal of this member.

Reply:
have done, thanks!